packages/salesforcedx-vscode/CHANGELOG.md

```diff
@@ -1,3 +1,15 @@
+# 48.21.0 - June 18, 2020
+
+## Added
```

Comment:
@sfsholden we can now mention the work on Mobile Preview for LWC components https://github.com/forcedotcom/salesforcedx-vscode/pull/2266

Reply:
@lcampos Cool, I will add that entry back in.